operational/flexera/msp/parent_child_group_sync/flexera_msp_parent_child_group_sync.pt

```diff
@@ -0,0 +1,1253 @@
+name "MSP Parent to Child Group Sync"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy synchronizes user groups, membership, and permissions from MSP Parent org to user groups in MSP Child orgs. See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/flexera/msp/parent_child_group_sync/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+category "Operational"
+severity "low"
+tenancy "single"
```

Comment:
Outdated concept. "tenancy" statements should be removed since we're moving away from multiple projects anyway. It also always defaults to single if you have a credentials block regardless.